shared/wallets/common/markdown-memo.tsx

```diff
@@ -38,16 +38,14 @@ const MarkdownMemo = (props: Props) =>
       style={Styles.collapseStyles([props.style, styles.container])}
     >
       {!props.hideDivider && <Kb.Divider vertical={true} style={styles.quoteMarker} />}
-      <Kb.Markdown
-        style={styles.memo}
-        styleOverride={{
-          ...(isMobile ? styleOverride : {}),
-          ...props.styleOverride,
-        }}
-        allowFontScaling={true}
-      >
-        {props.memo}
-      </Kb.Markdown>
+      <Kb.Text type="Body" style={styles.memo}>
+        <Kb.Markdown
+          styleOverride={Styles.collapseStyles([isMobile ? styleOverride : {}, props.styleOverride])}
```

Comment:
this is incorrect. styleOverride is a markdown specific concept. its not a regular style and passing collapseStyles to it isn't what you want. i think the real issue is you actually do want a style override passed in even in the mobile case so you can do that and remove the wrapping text here

Reply:
I don't think I want a style override, since the style overrides are for the children, not for the root node. Originally I investigated using that route, but the markup generated wouldn't allow that to matter since the mobile case has no wrapping node other than the wrapper we provide here.

Reply:
either way its invalid to pass collapseStyles to styleOverride
its been awhile since i looked at this component but i believe all `Text` node creation should flow through that override and if not thats likely a bug

Reply:
> either way its invalid to pass collapseStyles to styleOverride

Yes I pushed a fix

> all Text node creation should flow through that override

It does, but there is no root level text node created on mobile at all. This `Kb.Markdown` gets expanded directly to a list of nodes via `reactOutput`.

<img width="433" alt="Screen Shot 2019-10-01 at 6 22 09 PM" src="https://user-images.githubusercontent.com/59594/66011542-6dbfb700-e478-11e9-9077-d05e2a91ea06.png">
<img width="232" alt="Screen Shot 2019-10-01 at 6 22 20 PM" src="https://user-images.githubusercontent.com/59594/66011548-70baa780-e478-11e9-9421-f71dd66feeea.png">

Reply:
sure i mean those children should be using the styleOverride based on their types so should pull in whatever styling you need (but its been awhile since i looked at this)

Reply:
They are pulling in the style override. But we specifically need the children to be stacked vertically, which is controlled by the parent node. So we need to be able to style whatever their parent node is.

Another alternative that could work is to have a `shouldWrapOnMobile` flag that gives us the wrapper on mobile when desired, so that we pull in the style there.